### DISCUSSION TITLE
请问官网文档的代码编辑是用什么组件实现的

### DISCUSSION THREAD

**Discussion by easy-do** [DouyinFE/semi-design#667] [Q&A] [answered]
纯新手，我是服务端开发，最近自己研究做个小应用，前端用的是semi，正好遇到需要有输入代码的输入框，请问官网文档使用什么实现的呀，真好看。

**Comment by pointhalo** [comment, marked as answer]
monaco-editor  + react-live + 自定义主题 封装

https://github.com/DouyinFE/semi-design/discussions/203

**Reply by easy-do** [reply]
3q
